src/App.js

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
-import './App.css';
+// eslint-disable-next-line sort-imports
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import SideNavbar from 'components/SideNavbar/SideNavbar';
+import chatPage from 'pages/chatPage';
+import historyPage from 'pages/historyPage';
+import keywordPage from 'pages/keywordPage';
+import myPage from 'pages/myPage';
```

Comment:
페이지도 컴포넌트입니다... 다 수정하셔야겠네요.

```suggestion
import ChatPage from 'pages/ChatPage';
import HistoryPage from 'pages/HistoryPage';
import KeywordPage from 'pages/KeywordPage';
import MyPage from 'pages/MyPage';
```

Reply:
pages도 컴포넌트 메모메모...

Reply:
용어가 좀 애매하긴 한데... 중요한 것은 리액트 컴포넌트 형태로 만들어지는 모든 것들은 PascalCase 사용하는게 좋습니다 :)